src/capi_frontend/inferencerequest.cpp

```diff
@@ -60,6 +60,13 @@ Status InferenceRequest::setOutputBuffer(const char* name, const void* addr, siz
     }
     return it->second.setBuffer(addr, byteSize, bufferType, deviceId);
 }
+Status InferenceRequest::removeOutputBuffer(const char* name) {
+    auto it = outputs.find(name);
```

Comment:
Does it make sense to check `name` pointer validity first?

Reply:
It's checked at capi.cpp